sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_version.py

```diff
@@ -3,4 +3,4 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
-VERSION = "1.0.0b5"
+VERSION = "1.0.0"
```

Comment:
You PR says core and identity, for PR like this, I really want this to be just core and identity